src/components/graph/Graph.jsx

```diff
@@ -664,6 +664,8 @@ export default class Graph extends React.Component {
     const svgStyle = {
       height: this.state.config.height,
       width: this.state.config.width,
+      'background-color': this.state.config.backgroundColor,
```

Comment:
We could potentially expose the full object `style` and simply spread it on the `svgStyle`. What do you think about this? @LonelyPrincess @antoninklopp @terahn?

My suggestion would be something like a `style` object at the root config level `this.state.config.style`, that we could simply feed to the `svgStyle`.

```javascript
// I would just keep width/height segregated into their own props for retro-compatibility purposes
const svgStyle = {
  height: this.state.config.height,
  width: this.state.config.width,
  ...this.state.config.style,
};
```

Reply:
I think that this is definitely something that the configuration could provide

Reply:
What do you think, @henri-edinb? Are you up to do the change? This would be a much more meaningful contribution. Clients will be able to pass in any style to apply to the background.

Reply:
to be honest Im quite new to react so Im not sure exactly what needs to be done. setting `const svgStyle = {
  height: this.state.config.height,
  width: this.state.config.width,
  ...this.state.config.style,
};`

Reply:
I'm not able to give the proper guidance at the moment @henri-edinb. If someone can chime in please feel free to help @antoninklopp. What we're looking at here is:
* Add a new config object `config.style` which is an optional Object. By default, it should be empty in the default graph config.
* After that, we can perform the change that Henri is mentioning, spreading the `style` object on the `svgStyle`.
* Adjustments in the `Sandbox.js` might be required, I might be able to help with that later on